core/src/main/resources/schema/app/20220727094806_CB-17015_Add_InstanceStorageSize_to_Template.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-17015_Add_InstanceStorageSize_to_Template
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE template ADD COLUMN IF NOT EXISTS instancestoragesize INTEGER;
+UPDATE template SET instancestoragesize = 0 WHERE instancestoragesize IS NULL;
```

Comment:
Do you have stats on how many records are affected?
Please avoid setting defaults for new fields (as well as NON NULL constraints). Solve it from backend code.